deis-workflow-manager/manifests/deis-workflow-manager-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: deis-workflow-manager
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  type: LoadBalancer
+  selector:
+    app: deis-workflow-manager
+  ports:
+    - name: https
+      port: 8443
```

Comment:
If we do opt to make this a "routable application" then the service port should be 80.  The container port can still be whatever.